crowdin.yml

```diff
@@ -0,0 +1,7 @@
+project_id_env: CROWDIN_PROJECT_ID
```

Comment:
I went ahead and added back the crowdin.yml file so that I could easily add translations. There is another ticket to use the Crowdin API, so that we will have more granular control on getting the data and what we do with it.